### DISCUSSION TITLE
請問一下，"+"在sendkeys裡面要如何表示?

### DISCUSSION THREAD

**Discussion by ClayWCL** [shuax/MouseInc.Settings#395] [Q&A] [answered]
我有一個程式的熱鍵是Alt+Shift+"+"，
當中有包含"+"，在sendkeys的部分我嘗試用plus表示，結果是輸出"="，
請問一下，"+"在sendkeys裡面要如何表示呢?
謝謝

**Comment by shuax** [comment]
去掉shift，或者把plus改成=试试

**Reply by ClayWCL** [reply]
用=還是輸出=
"+" 真的不知道要用什麼方式表達...

**Comment by shuax** [comment]
我这里测试没问题，可能你那个软件不兼容吧。
![image](https://user-images.githubusercontent.com/6940583/189519382-3066c184-790c-4d28-8d48-f8aaa149e983.png)
![image](https://user-images.githubusercontent.com/6940583/189519392-ca0d7b6d-e0a3-4b9a-8d13-6ab455aa29b2.png)

**Comment by shuax** [comment, marked as answer]
你那个加该不会是小键盘的加吧，得用`Num Plus`

**Reply by ClayWCL** [reply]
沒錯就是數字鍵那裡的加號，我原本嘗試過Numplus，結果原來是「Num Plus」！
太好了，問題解決了感謝！
